admin/pre_reg/forms.py

```diff
@@ -0,0 +1,38 @@
+import itertools
+
+from django import forms
+
+from admin.pre_reg.utils import get_prereg_reviewers
+PREREG_REVIEWERS = itertools.chain(((None, 'None'), ), get_prereg_reviewers())
+
+class DraftRegistrationForm(forms.Form):
+
+    proof_of_publication = forms.ChoiceField(
+        label="Proof of publication",
+        choices=(
+            ("not_submitted", "Published Article Not Yet Submitted"),
+            ("submitted", "Published Article Submitted"),
+            ("under_review", "Published Article Under Review"),
+            ("approved", "Published Article Approved"),
+            ("rejected", "Published Article Rejected"),
+        )
+    )
+    payment_sent = forms.BooleanField(
+        label="Payment sent"
+    )
+    assignee = forms.ChoiceField(
+        label="Assignee",
+        choices=PREREG_REVIEWERS
+    )
+
+    notes = forms.CharField(
+        label="Notes",
+        widget=forms.Textarea(
+            attrs={
+                'style': 'height: 50px;'
```

Comment:
This would be better if it were in css. At least when someone tries to figure out how to change it.